### ISSUE TITLE
Accessibility: Table sort icons with role="presentation" must not have aria-labels

### ISSUE THREAD

**Issue by Ke1sy** [ant-design/ant-design#40412]
### Reproduction link

[https://ant.design/~demos/components-table-demo-head](https://ant.design/~demos/components-table-demo-head)

### Steps to reproduce

Open the example and inspect the source code with dev tools. You will see the next code for table column sorter icons:
```
<span role="presentation" aria-label="caret-up" class="anticon anticon-caret-up ant-table-column-sorter-up">...</span>
<span role="presentation" aria-label="caret-down" class="anticon anticon-caret-down ant-table-column-sorter-down active">...</span> 
```
Analyze the page accessibility with Lighthouse or Axe accessibility plugin. 

### What is expected?

Element with role="none" or role="presentation" must not have one of the following characteristics:
- A global ARIA attribute.

Error description: [Deque university](https://dequeuniversity.com/rules/axe/4.2/presentation-role-conflict).

### What is actually happening?

The accessibility error: Each ARIA `role` supports a specific subset of `aria-*` attributes. Mismatching these invalidates the `aria-*` attributes. Aria-label attribute cannot be used with role "null".
Error description: [Deque university](https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?utm_source=lighthouse&utm_medium=devtools).
Impact: serious.

| Environment | Info |
| --- | --- |
| antd | 5.1.6 |
| React | 18.0.0 |
| System | Windows 10 |
| Browser | Chrome 109.0.5414.119 |

---
Proposed solution: remove aria-label for the sort icons
